### ISSUE TITLE
More leaderboard issues

### ISSUE THREAD

**Issue by tentamens** [Coflnet/CtwApi#33]
When I call Stats api it says I have 1025 XP but on the leaderboard all time it says I have 820 XP

**Comment by Ekwav** [comment]
may I ask you to include the json response you got here?

**Comment by tentamens** [comment]
For what, the Stats api?

**Comment by tentamens** [comment]
`[Stat[userId=847e326b-c39b-484e-9621-c46962d04c04, statName=exp, value=1025], Stat[userId=847e326b-c39b-484e-9621-c46962d04c04, statName=images_uploaded, value=25]]` This is stats response

**Comment by Ekwav** [comment]
I get 
```json
[
  {
    "user": {
      "userId": "69b550f8-dbf3-4936-9fa6-1a1d63a82751",
      "name": "tentamens ",
      "avatar": "v;M.h|"
    },
    "score": 2000
  },
  {
    "user": {
      "userId": "847e326b-c39b-484e-9621-c46962d04c04",
      "name": "tentamens ",
      "avatar": "SJiksI"
    },
    "score": 1230
  },
  {
    "user": {
      "userId": "edf6866f-442d-4a0c-b70a-96e2dfd12e56",
      "name": "Unknown",
      "avatar": null
    },
    "score": 505
  }
]
```
for the overall leaderboard
and for the stats of `847e326b-c39b-484e-9621-c46962d04c04`
```json
[
  {
    "userId": "847e326b-c39b-484e-9621-c46962d04c04",
    "statName": "current_offset",
    "value": 4
  },
  {
    "userId": "847e326b-c39b-484e-9621-c46962d04c04",
    "statName": "exp",
    "value": 1230
  },
  {
    "userId": "847e326b-c39b-484e-9621-c46962d04c04",
    "statName": "images_uploaded",
    "value": 26
  }
]
```
That matches up exactly (I assume you uploaded one image changing the exp and images_uploaded)

**Comment by Ekwav** [comment]
I assume this is fixed with #31
